meta/Search.php

```diff
@@ -75,14 +77,15 @@ public function addSchema($table, $alias = '') {
             throw new StructException('schema missing', $table);
         }
 
-        if($this->schemas &&
-            (
-                $schema->isLookup() ||
-                reset($this->schemas)->isLookup()
-            )
-        ) {
-            throw new StructException('nolookupmix');
-        }
+        // FIXME is the mixing still relevant?
```

Comment:
This check probably needs to be more clever, though I am not sure if it's feasible. Whenever multiple schemas are selected, it would need to be ensured that they have PID data. But maybe we can skip this checking and leave it to the user to do sensible aggregations.